packages/arb-token-bridge-ui/src/components/TransactionsTable/TransactionsTableWithdrawalRow.tsx

```diff
@@ -210,7 +214,7 @@ export function TransactionsTableWithdrawalRow({
   }
 
   return (
-    <tr className={`text-sm text-dark ${className}`}>
+    <tr className={`bg-cyan text-sm text-dark even:bg-white ${className}`}>
```

Comment:
didn't see the "Confirmed" status on the mockups, but lots of "Pending" status.
Can you please align with Val to ensure the "Confirmed"'s styles have the expected design?
<img width="659" alt="image" src="https://user-images.githubusercontent.com/13184582/217239256-49404197-1c50-4306-a6b9-547bcd6cdfa0.png">